spec/features/consumer/shopping/shopping_spec.rb

```diff
@@ -294,19 +298,21 @@
 
         click_add_to_cart variant, 10
 
-        expect(page).to have_field "variants[#{variant.id}]", with: '10'
+        within_variant(variant) do
+          expect(page).to have_content "10 in cart"
+        end
       end
 
       it "alerts us when we enter a quantity greater than the stock available" do
         variant.update on_hand: 5
         visit shop_path
 
-        
-        accept_alert 'Insufficient stock available, only 5 remaining' do
-          fill_in "variants[#{variant.id}]", with: '10'
-        end
+        click_add_to_cart variant, 5
 
-        expect(page).to have_field "variants[#{variant.id}]", with: '5'
+        within_variant(variant) do
+          expect(page).to have_content "5 in cart"
+          expect(page).to have_button "＋", disabled: true
```

Comment:
is this that special plus sign? if yes, this should be extracted to a method to make it clear.
def large_unicode_plus_sign
    "＋"
end

and then
expect(page).to have_button large_unicode_plus_sign, disabled: true

Reply:
this would then be used in all occurrences of the special plus sign in the specs

Reply:
Yes, agree. I will do that.